DashAI/back/models/classes/numerical_wrapper_for_text.py

```diff
@@ -2,11 +2,11 @@
 
 from sklearn.feature_extraction.text import CountVectorizer
 
-from DashAI.back.models.classes.sklearn_like_model import SklearnLikeModel
-from DashAI.back.models.classes.text_classification_model import TextClassificationModel
+from DashAI.back.models.scikit_learn.sklearn_like_model import SklearnLikeModel
```

Comment:
Why we don't import this on DashAI/back/models/__init__.py?

Reply:
Ok